### PR TITLE
FIX: responsiveness for images

### LINKED ISSUE #27040: Images no longer responsive
### Preconditions (*)

Since Magento 2.3.4 (commit https://github.com/magento/magento2/commit/d0ad63cca0b8e29b86b29c72608d034658d244a5) the height auto has been removed.
this clearly was not tested as all images on on the Magento front-end break and are distorted thanks to the size.

### Steps to reproduce (*)

View any image in Magento 2.3.4.
for example the logo.

### Expected result (*)

Images should be responsive.

### Actual result (*)

_Same as Expected result_

### PR DESCRIPTION
**Pull request by GrimLink**
### Description (*)

Since Magento 2.3.4 (commit d0ad63c) the height auto has been removed.
this clearly was not tested as all images on on the Magento front-end break and are distorted thanks to the size.

### Fixed Issues (if relevant)

Fixes: https://github.com/magento/magento2/issues/27040

### Manual testing scenarios (*)

Setup a Magento site with version 2.3.4 or higher
Create a CSS change containing the following CSS:

```css
img {
  height: auto;
}
```

This will fix the issue at hand.

The change off course is fixed at the reset styles.
Where in the previous version this solution was.
Before the change from commit https://github.com/magento/magento2/commit/d0ad63cca0b8e29b86b29c72608d034658d244a5

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by krzksz — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @krzksz, thank you for the review.
[ENGCOM-6999](https://jira.corp.magento.com/browse/ENGCOM-6999) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
@GrimLink thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

File: lib/web/css/source/lib/_resets.less [hunk 1]
```diff
@@ -51,14 +51,17 @@
         border: 0;
```
Comment (engcom-Alfa) [inline lib/web/css/source/lib/_resets.less:53]: please move img styles to this block, as they were overridden
```
      max-width: 100%;
      height: auto;
```

**Review by engcom-Alfa — CHANGES_REQUESTED** [review]
@GrimLink Please, see my review comment. Thanks!

File: lib/web/css/source/lib/_resets.less (hunk 1 above)
Comment (GrimLink) [reply lib/web/css/source/lib/_resets.less:53]: Done, Yes it better to have the img styles merged under 1 selector.

**Review by krzksz — APPROVED** [review]
Good call, looks great!

**Comment by magento-engcom-team** [comment]
Hi @krzksz, thank you for the review.
[ENGCOM-6999](https://jira.corp.magento.com/browse/ENGCOM-6999) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Pull request merged**